projects/backend/controllers/appsRendering.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express'
+import fetch from 'node-fetch'
+
+const fetchRenderedArticle = async (url: string) => {
+    const response = await fetch(url, {
+        headers: {
+            Accept:
+                'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+        },
+    })
+    const responseBody = await response.text()
```

Comment:
I think it should be possible to [pipe the body](https://github.com/node-fetch/node-fetch#streams) of the response from apps-rendering straight into the body of the response you're returning to Editions. In theory this would speed up the request and save on memory overhead (I think `response.text()` loads the HTML body into memory?).

This is an optimisation that's probably unnecessary here because this is a temporary solution for development - but we may want to use it when we wire up the direct requests to AR down the road?

Reply:
certainly looks neater.